crates/bevy_render/src/view/visibility/mod.rs

```diff
@@ -47,6 +47,30 @@ pub enum Visibility {
     Visible,
 }
 
+impl Visibility {
+    /// function converting [Visibility] enum to Result<bool, String>
+    /// - returns `Ok(true)` if `Visibility::Visible`
+    /// - returns `Ok(false)` if `Visibility::Hidden`
+    /// - returns `Err()` if `Visbility::Inherited`
+    pub fn is_visible(self) -> Result<bool, String> {
```

Comment:
The error type here should either be `()` or ideally a custom Zero-Sized Type that implements the `Error` trait :) `thiserror` is useful to help implement the trait. Strings require allocations, can't be matched on and there's not much information there.